Frends.ServiceBus.Read/Frends.ServiceBus.Read/Definitions/Options.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.ServiceBus.Read.Definitions;
+
+/// <summary>
+/// Option parameters 
+/// </summary>
+public class Options
+    {
+    /// <summary>
+    /// Timeout in seconds.
+    /// </summary>
+    /// <example>60</example>
+    [DefaultValue("60")]
+    public int TimeoutSeconds { get; set; }
+
+    /// <summary>
+    /// Should the service bus connection (MessagingFactory) be cached. This speeds up the execution as creating a new connection is slow by keeping the connection open in the background, a single namespace is limited to 1000 concurrent connections.
+    /// </summary>
+    /// <example>true</example>
+    [DefaultValue(true)]
+    public bool UseCachedConnection { get; set; }
+
+    /// <summary>
+    /// How the body is expected to be serialized.
+    /// </summary>
+    /// <example>Stream</example>
+    [DefaultValue(BodySerializationType.Stream)]
+    public BodySerializationType BodySerializationType { get; set; }
+
+    /// <summary>
+    /// What encoding the message contents is expected to be in.
+    /// </summary>
+    /// <example>UTF8, UTF32, ASCII, Unicode, Latin1, BigEndianUnicode</example>
+    [DefaultValue(MessageEncoding.UTF8)]
+    public MessageEncoding DefaultEncoding { get; set; }
+
+    /// <summary>
+    /// Should the existence of the message source be checked and created if it does not exist.
+    /// </summary>
+    /// <example>False</example>
+    [DefaultValue(false)]
+    public bool CreateQueueOrTopicIfItDoesNotExist { get; set; }
+
+    /// <summary>
+    /// Idle interval after which the queue or topic+subscription is automatically deleted. See TimeFormat to select Minutes/Hours/Days/Never. The minimum duration is 5 minutes and contains converter e.g. 61 minutes = 1 hour 1 minute.
+    /// </summary>
+    /// <example>5</example>
+    [UIHint(nameof(CreateQueueOrTopicIfItDoesNotExist), "", true)]
+    [DefaultValue("5")]
+    public int AutoDeleteOnIdle { get; set; }
```

Comment:
And same here, you can make multi-line comment. No need to keep all on the same line.

Reply:
I think it's cleaner to keep these in 1 line because you can word wrap these lines in Visual Studio when using smaller resolution.